pyanaconda/modules/storage/nvdimm/nvdimm.py

```diff
@@ -101,6 +101,11 @@ def get_devices_to_ignore(self):
         devices_to_ignore = set()
 
         for ns_name, ns_info in nvdimm.namespaces.items():
+            if ns_info.blockdev is None:
+                log.debug("%s - NVDIMM namespace block device information "
```

Comment:
I would use `%s will be skipped - ` to follow the pattern of the messages below.